src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -16,13 +17,14 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp3.1'">
     <!-- Instrumented libraries -->
     <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.0.0" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Abstractions" Version="2.0.0" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Abstractions" Version="2.0.0" />
```

Comment:
Ouch. I remember the discussion on last SIG that we should not do such stuff. Am I correct or is this some different case?

Reply:
This line is my error, I removed that from the wrong package :|